config/webpack.dist.config.js

```diff
@@ -52,12 +53,6 @@ module.exports = {
     new webpack.DefinePlugin({
       'process.env': { NODE_ENV: JSON.stringify(nodeEnv) },
     }),
-    // Disabling until https://github.com/fritz-c/react-image-lightbox/issues/88 is resolved
-    // Causing an issue within main application
-    // ***** 2/28/2019 UPDATE *****
-    // Re-enabling UglifyJs since conflict seems to be resolved. Keeping original comment above in case issue re-surfaces. -- Juan Fabrega
-    new webpack.optimize.UglifyJsPlugin(),
```

Comment:
Is this being handled somewhere else now?

Reply:
handled now with terser-webpack-plugin